### PR TITLE
Allow change number of beats

### PR DESCRIPTION
**Pull request by FelipeSVidal**
Fix playNote, first note was't played in first time and playing previous note.

### CODE REVIEW

File: src/Components/DrumPad/index.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const DrumPad = ({ count, layout, swapBeat }) => {
               <td
                 key={beatNum}
                 onClick={() => {
-                  console.log({ rowNum, beatNum });
+                  console.log({ rowNum, beatNum, hasBeat });
```
Comment (rgavinc) [inline src/Components/DrumPad/index.js]: - [ ] Can you remove this console.log entirely? I accidentally left it in after I was debugging something :)

File: src/pages/index.js [hunk 2]
```diff
@@ -40,10 +39,29 @@ const Home = ({ originalLayout, error }) => {
...
+    if (newLayout[0].beats.length === 16) return;
+    newLayout.forEach(e => {
+      e.beats.push(false);
+    });
+    setLayout(newLayout);
+  };
+  const rmvBeat = () => {
+    const newLayout = [...layout];
+    if (newLayout[0].beats.length === 4) return;
+    newLayout.forEach(e => {
```
Comment (rgavinc) [inline src/pages/index.js]: - [ ] Can you change `e` to `row` so we know what that variable is?

File: src/pages/index.js [hunk 3]
```diff
@@ -40,10 +39,29 @@ const Home = ({ originalLayout, error }) => {
     setLayout(newLayout);
   };
 
+  const addNewBeat = () => {
+    const newLayout = [...layout];
+    if (newLayout[0].beats.length === 16) return;
+    newLayout.forEach(e => {
```
Comment (rgavinc) [inline src/pages/index.js]: - [ ] Can you change `e` to `row` so we know what that variable is?

File: src/Components/DrumPad/index.js [hunk 4]
```diff
@@ -34,7 +34,7 @@ const DrumPad = ({ count, layout, swapBeat }) => {
               <td
                 key={beatNum}
                 onClick={() => {
-                  console.log({ rowNum, beatNum });
+                  //console.log({ rowNum, beatNum, hasBeat });
```
Comment (rvvergara) [inline src/Components/DrumPad/index.js]: Instead of commenting this out, I suggest we should completely delete this line :)

**Comment by rvvergara** [comment]
Hi @FelipeSVidal. I'm merging this PR :) Let's just delete those commented code later on. Ok? Great job bro!

**Pull request merged**